test/e2e/cron_test.go

```diff
@@ -43,7 +43,6 @@ func (s *CronSuite) TearDownSuite() {
 
 func (s *CronSuite) TestBasic() {
 	s.Run("TestBasic", func() {
-		s.T().Parallel()
```

Comment:
These don't do anything, and as this was the only attempt in the code to parallelise tests, we could remove the `-parallel` flag on the test suite.

I'd like to find a way to reintroduce parallel testing in the future though, so let's not do that.